scripts/pi-hole/js/auditlog.js

```diff
@@ -1,137 +1,163 @@
+          url =
+            '<a href="queries.php?domain=' +
+            printdomain +
+            '">' +
+            printdomain +
+            "</a> (wildcard blocked)";
+          adtable.append(
+            "<tr> <td>" +
+              url +
+              "</td> <td>" +
+              data.top_ads[domain] +
+              '</td> <td> <button class="text-orange text-nowrap"><i class="fa fa-balance-scale"></i> Audit</button> </td> </tr> '
+          );
+        } else {
+          url =
+            '<a href="queries.php?domain=' +
+            printdomain +
+            '">' +
+            printdomain +
+            "</a>";
+          adtable.append(
+            "<tr> <td>" +
+              url +
+              "</td> <td>" +
+              data.top_ads[domain] +
+              '</td> <td> <button class="text-green text-nowrap"><i class="fas fa-check"></i> Whitelist</button> <button class="text-orange text-nowrap"><i class="fa fa-balance-scale"></i> Audit</button> </td> </tr> '
+          );
         }
+      }
+    }
 
-        $("#domain-frequency .overlay").hide();
-        $("#ad-frequency .overlay").hide();
-        // Update top lists data every second
-        setTimeout(updateTopLists, 1000);
-    });
+    $("#domain-frequency .overlay").hide();
+    $("#ad-frequency .overlay").hide();
+    // Update top lists data every second
+    setTimeout(updateTopLists, 1000);
+  });
 }
 
-
-function add(domain,list) {
-    var token = $("#token").html();
-    $.ajax({
-        url: "scripts/pi-hole/php/add.php",
-        method: "post",
-        data: {"domain":domain, "list":list, "token":token}
-    });
+function add(domain, list) {
+  var token = $("#token").html();
+  $.ajax({
+    url: "scripts/pi-hole/php/add.php",
+    method: "post",
+    data: { domain: domain, list: list, token: token }
+  });
 }
 
 $(document).ready(function() {
-
-    // Pull in data via AJAX
-    updateTopLists();
-
-    $("#domain-frequency tbody").on( "click", "button", function () {
-        var url = ($(this).parents("tr"))[0].innerText.split("	")[0];
-        if($(this).context.innerText === " Blacklist")
-        {
-            add(url,"audit");
-            add(url,"black");
-            $("#gravityBtn").prop("disabled", false);
-        }
-        else
-        {
-            auditUrl(url);
-        }
-    });
-
-    $("#ad-frequency tbody").on( "click", "button", function () {
-        var url = ($(this).parents("tr"))[0].innerText.split("	")[0].split(" ")[0];
-        if($(this).context.innerText === " Whitelist")
-        {
-            add(url,"audit");
-            add(url,"white");
-            $("#gravityBtn").prop("disabled", false);
-        }
-        else
-        {
-            auditUrl(url);
-        }
-    });
+  // Pull in data via AJAX
+  updateTopLists();
+
+  $("#domain-frequency tbody").on("click", "button", function() {
+    var url = $(this)
+      .parents("tr")[0]
+      .textContent.split("	")[0];
```

Comment:
BTW here and later we should probably see if using `\t` works and switch to that.